lib/oli_web/components/delivery/students/students.ex

```diff
@@ -1292,8 +1292,8 @@ defmodule OliWeb.Components.Delivery.Students do
         end),
       low_proficiency:
         Enum.count(students, fn student ->
-          (Metrics.proficiency_range(student.overall_proficiency) == "Low" ||
-             is_nil(student.overall_proficiency)) and is_learner_selected(student, filter_by)
+          student.overall_proficiency == "Low" ||
+            (is_nil(student.overall_proficiency) and is_learner_selected(student, filter_by))
```

Comment:
I'm not sure why we are checking if `student.overall_proficiency` is nil here, I don't see any way in which it could be null since the values default to "Not enough data" on initialization. I tried to leave this logic in tact though because it seemed to break some unit tests and I also want to limit the changes here if possible